### PR TITLE
Add Webpack and basic component

### PR DESCRIPTION
**Pull request by KimarieHB**
Add react, webpack and dependencies. Renders very basic hello world.

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const getBonusItem = (title, callback) => {
 
 // Get song to be played upon selection
 const getSong = (trackName, callback) => {
-  Soundtrack.findOne({ 'bonus_info.tracklist.name': trackName}, {'bonus_info.tracklist.$': 1, _id: 0}, (err, data) => {
+  Soundtrack.findOne({ 'bonus_info.tracklist.name': trackName }, { 'bonus_info.tracklist.$': 1, _id: 0 }, (err, data) => {
```
Comment (KimarieHB) [inline database/index.js:41]: I just added spaces here for consistency.

File: client/dist/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <title>HumbleBundle</title>
+    </head>
+    <body>
+      <div id='bonuses'></div>
+      <script type='text/javascript' src='bundle.js'></script>
+    </body>
+  </html>
```
Comment (redperezz) [inline client/dist/index.html:10]: ✓

File: client/src/index.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  }
+
+  render() {
+    return (
+      <div>Hello World!</div>
+    )
+  }
+}
+
+ReactDOM.render(<App />, document.getElementById('bonuses'));
```
Comment (redperezz) [inline client/src/index.jsx:16]: ✓

File: database/index.js (hunk 1 above)
Comment (redperezz) [reply database/index.js:41]: ok yeah that's a bit more readable ✓

**Review by redperezz — APPROVED** [review]
Everything looks good!

**Pull request merged**